src/react-apps/applications/runtime/src/containers/withFormElementHandlers.tsx

```diff
@@ -115,7 +115,7 @@ export const formComponentWithHandlers = (WrappedComponent: React.ComponentType<
       }
       const simpleBinding = component.dataModelBindings.simpleBinding;
       const type = component.type;
-      return simpleBinding && type !== 'Checkboxes' && type !== 'RadioButtons' && type !== 'FileUpload';
```

Comment:
@lorang92 why was this set in the first place?

Reply:
No idea, this was before my time.